sdk/purview/Azure.Analytics.Purview.Administration/src/Customizations/PurviewCollection.cs

```diff
@@ -9,6 +9,7 @@ namespace Azure.Analytics.Purview.Administration
 {
     [CodeGenClient("CollectionsClient")]
     [CodeGenSuppress("PurviewCollection", new Type[] { typeof(Uri), typeof(string), typeof(TokenCredential), typeof(PurviewAccountClientOptions)})]
+    [CodeGenSuppress("PurviewCollection", new Type[] { typeof(Uri), typeof(string), typeof(TokenCredential)})]
```

Comment:
It looks like this change suppresses the subclient constructor manually.  What is our plan to fix the output of the generator so that it doesn't generate constructors for subclients?

Reply:
@pshao25, let's make sure we have an issue filed to track this work, and that it is blocking GA.

Reply:
I would expect that if we provide a parent, e.g. `[CodeGenClient("CollectionsClient", ParentClient = typeof(PurviewAccountClient))]`, then we should be able to remove all `CodeGenSuppress` attributes for this type and also get rid of manually added constructor below.

Reply:
This is just because the force update from newly-merged code. I update it again,